scripts/setup-sources.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# A quick and dirty script to prepare the source code needed for
+# the apalache bench tests
+
+set -euf -o pipefail
+
+latest="0.7.0"
+devl_dir="${HOME}/devl"
+
+mkdir -p "$devl_dir"
+git clone git@github.com:informalsystems/apalache.git "$devl_dir/apalache" > /dev/null
+
+for v in "0.5.1" "0.6.0" "0.7.0"
+do
+    echo "preparing source code for apalache version ${v}"
+    version_dir="${HOME}/devl/apalache-${v}"
+    cp -r "${devl_dir}/apalache" "$version_dir"
+    pushd "$version_dir"
+    git fetch
+    if [[ "$v" == "$latest" ]]
+    then
+        branch="origin/unstable"
```

Comment:
Is this the correct behavior for when we are testing the `latest` version, for which we have not yet cut a release? (Perhaps it should be called "unreleased"?).

Reply:
The behavior is correct. Shall we just call it `unstable`, as `latest` can be indeed ambiguous?